app/services/attempts_api/tracker_events.rb

```diff
@@ -11,6 +11,11 @@ def email_and_password_auth(success:)
       )
     end
 
+    # A user is transferred to the in-person proofing workflow
```

Comment:
Could this comment be more specific?  Maybe, `A user completes the online portion of the in-person proofing process` or `A user becomes able to visit the post office for in-person proofing`.

Reply:
updated in https://github.com/18F/identity-idp/pull/12093/commits/23872c5ee580b9e75b3d1466547ed5f4bc9180f3